index.js

```diff
@@ -0,0 +1,3 @@
+const nombreUsuario = prompt('Ingrese su nombre');
+
+alert(`Hola ${nombreUsuario}`);
```

Comment:
Acá sería mejor no usar interpolación de strings.
```suggestion
alert('Hola' + nombreUsuario);
```